### PR TITLE
Add user document of time zone 

### PR DESCRIPTION
**Pull request by Alima777**

### CODE REVIEW

File: docs/UserGuide/API/Time-zone.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# Time zone
+
+When a client connects to the iotdb server, it can specify the time zone to be used for this connection. If not specified, the default time zone value is the time zone of the client.
```
Comment (liujiajun) [inline docs/UserGuide/API/Time-zone.md:24]: IoTDB?

Comment (Alima777) [reply docs/UserGuide/API/Time-zone.md:24]: Okay~ Thank you

File: docs/UserGuide/API/Time-zone.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+When a client connects to the IoTDB server, it can specify the time zone to be used for this connection. If not specified, the default time zone is the one of the client.
+
+The time zone can be set in both JDBC and session native interface connections. The usage is as follows:
+
+```java
+JDBC: (IoTDBConnection) connection.setTimeZone("+08:00");
+
+Session: session.setTimeZone("+08:00");
+```
```
Comment (HTHou) [inline docs/UserGuide/API/Time-zone.md:32]: Should we tell users the time zone usage in CLI here?
For example, we can set the time zone by the following statement.
```sql
set time_zone=+04:00
```

Comment (Alima777) [reply docs/UserGuide/API/Time-zone.md:32]: All right.

**Pull request merged**